satpy/readers/viirs_sdr.py

```diff
@@ -27,15 +27,23 @@
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
 
 """Interface to VIIRS SDR format
+**********************************
```

Comment:
Does this render properly in the sphinx API docs?

Reply:
yes, see here:
http://rst.ninjs.org/?theme=nature#SW50ZXJmYWNlIHRvIFZJSVJTIFNEUiBmb3JtYXQKKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKgoKVGhpcyByZWFkZXIgaW1wbGVtZW50cyB0aGUgc3VwcG9ydCBvZiBWSUlSUyBTRFIgZmlsZXMgYXMgcHJvZHVjZWQgYnkgQ1NQUC4gSXQgaXMKY29tcHJpc2VkIG9mIHR3byBwYXJ0czoKCiAtIEEgc3ViY2xhc3Mgb2YgdGhlIFlBTUxGaWxlUmVhZGVyIGNsYXNzIHRvIGFsbG93IGhhbmRsaW5nIGFsbCB0aGUgZmlsZXMKIC0gQSBmaWxlaGFuZGxlciBjbGFzcyB0byBpbXBsZW1lbnQgdGhlIGFjdHVhbCByZWFkaW5nCgpGb3JtYXQgZG9jdW1lbnRhdGlvbjoKCiAtIGh0dHA6Ly9ucHAuZ3NmYy5uYXNhLmdvdi9zY2llbmNlL3NjaWVuY2Vkb2N1bWVudHMvMDgyMDEyLzQ3NC0wMDAwMS0wM19DREZDQlZvbElJSV9SZXZDLnBkZg==

Reply:
Yeah it is valid rst, but does sphinx like it and does it look bad in API pages.

Reply:
Any update on this? Does sphinx like having the "subject line" of the docstring having a section separator?

Reply:
Yep
![sphinx](https://user-images.githubusercontent.com/167802/54433321-d9844180-472b-11e9-8d90-908722f5e9f5.png)

Reply:
Cool